dlrs/main/lwc/jsconfig.json

```diff
@@ -1,12 +1,25 @@
 {
-  "compilerOptions": {
-    "experimentalDecorators": true
-  },
-  "include": ["**/*", "../../../.sfdx/typings/lwc/**/*.d.ts"],
-  "paths": {
-    "c/*": ["*"]
-  },
-  "typeAcquisition": {
-    "include": ["jest"]
-  }
+    "compilerOptions": {
```

Comment:
This file is still showing modified in your PR. Please revert this change.

Reply:
This should be aligned with what's on the main branch.